docs/source/api/python/dgl.data.rst

```diff
@@ -62,6 +62,12 @@ Datasets for node classification/regression tasks
     CornellDataset
     TexasDataset
     WisconsinDataset
+    HeterophilousGraphDataset
```

Comment:
There is no need to expose the base class to users. I suggest removing it.

Reply:
Done